src/components/studies/StudyList.tsx

```diff
@@ -176,6 +178,59 @@ const StudySublist: FunctionComponent<StudySublistProps> = ({
   )
 }
 
+const VerifyBanner: FunctionComponent<{isVerified: boolean | undefined}> = ({isVerified}) => {
+  const [displayVerifyBanner, setDisplayVerifyBanner] = React.useState<boolean>(!isVerified)
+
+  if (typeof isVerified === undefined) {
```

Comment:
You can display this conditionally in a parent (below) only if verified !== true. And then you don't need 'isVerified' param

Reply:
Oh got it, makes sense! Done in next commit